internal/server/iigointernal/orchestration.go

```diff
@@ -214,6 +219,9 @@ func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.C
 		return false, "President was not apointed by the Judge. Insufficient budget"
 	}
 
+	legislativeBranch.reset()
+	executiveBranch.reset()
```

Comment:
i very much don't like the reset pattern, why not create a new struct the next time with the right values?

Reply:
sure thing!